### DISCUSSION TITLE
Compile the source code error

### DISCUSSION THREAD

**Discussion by waterMeteors** [dbeaver/dbeaver#13323] [DBeaver Development]
hello everyone, I want to read the source code. when I clone the source code and running the "mvn package".It is always error that message is  like the picture
![image](https://user-images.githubusercontent.com/54666561/126623583-0486aa2e-d91d-43ab-9e43-fc69f3e58a43.png)

System environment：
  windows 64
java version : "1.8.0_131"
maven version : "Apache Maven 3.6.3"

if I want to run the source code ,what should I do

**Comment by LonwoLonwo** [comment]
Hello @waterMeteors 
I can download files from here https://dbeaver.io/eclipse-repo/plugins/
And "mvn package" works fine for me.
Please try again, it can be a temporary failure.
Also it can be some kind of proxy/firewall settings problem.
